python/hello-world/README.md

```diff
@@ -0,0 +1,24 @@
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/python/hello-world)
+
+# Python Hello World
+
+This example shows how to use Python on Vercel with Serverless Functions using the [Python Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python).
+
+## Demo
+
+https://python-hello-world.vercel.app/
+
+## Running Locally
+
+```bash
+npm i -g vercel
```

Comment:
```suggestion
pip install vercel
```

I think for python it would be cool to do use `pip`

https://pypi.org/project/vercel/

not sure how updated those are and what's out strategy here. So I leave it as an optional suggestion.